### PR TITLE
Blog post for LWIMIQ June 19, 2017

### PR DESCRIPTION
**Pull request by djberg96**
Last Week in ManageIQ blog post for June 19, 2017.

### CODE REVIEW

**Comment by cybette** [comment]
Thanks @djberg96, good one! Just a couple things:
- please add extension .md to filename (for it to be recognized as markdown)
- use your github id `djberg96` for author, we hope to have authors linked automatically in future.

**Comment by cybette** [comment]
@jntullo @imtayadeway @syncrou kindly have a look as well, thanks!

**Comment by djberg96** [comment]
Hi Carol,

I've updated the extensions (oops) and used "djberg96".

Thanks,

Dan

File: site/_posts/2017-06-18-The-UI-is-the-thing.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## Improved
+
+The Classic UI saw several improvements. Some highlights include:
+
+[Policy event assignment styling](https://github.com/ManageIQ/manageiq-ui-classic/pull/1543)
+
+[Sane default port for Lenovo](https://github.com/ManageIQ/manageiq-ui-classic/pull/1531)
+
+[Show last refresh time](Show last refresh time & date in provider summary screen)
```
Comment (JPrause) [inline site/_posts/2017-06-18-The-UI-is-the-thing.md]: Was this a reminder to add a URL

**Comment by jntullo** [comment]
Besides what @jprause said, LGTM! 👍

File: site/_posts/2017-06-18-The-UI-is-the-thing.md (hunk 1 above)
Comment (djberg96) [inline site/_posts/2017-06-18-The-UI-is-the-thing.md]: @JPrause Oops, fixed.

**Comment by JPrause** [comment]
LGTM

File: site/_posts/2017-06-18-The-UI-is-the-thing.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+---
+title: "Last Week in ManageIQ: The UI is the thing"
+author: Daniel Berger
```
Comment (cybette) [inline site/_posts/2017-06-18-The-UI-is-the-thing.md]: sorry I wasn't clear enough, I meant to use your github id here

File: site/_posts/2017-06-18-The-UI-is-the-thing.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+date: 2017-06-20
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hi all! [djberg96](https://github.com/djberg96) here again from the Providers team. This is my second Last Week in ManageIQ post, where I try to distill the week's highlights of ManageIQ development into a nice, digestable format.
```
Comment (cybette) [inline site/_posts/2017-06-18-The-UI-is-the-thing.md]: and here you can use your full name (probably preferred)

**Comment by djberg96** [comment]
@cybette Oops, fixed.

**Pull request merged**